src/utils/venue.ts

```diff
@@ -85,5 +90,8 @@ export const createJazzbar = (values: FormValues): Venue => {
     adultContent: values.adultContent || false,
     width: values.width ?? 40,
     height: values.width ?? 40,
+    // @debt Should these fields be defaulted like this? Or potentially undefined? Or?
+    iframeUrl: "",
+    logoImageUrl: "",
```

Comment:
[note] These needed to be added so that we could properly use `JazzbarVenue` as the return type here.